style.css

```diff
@@ -513,6 +522,133 @@ nav.active {
   transform: scale(1.1);
 }
 
+/* Window Modal Style */
+.overlay {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+  max-width: inherit;
+  max-height: inherit;
+  background-color: rgba(0, 0, 0, 0.4);
+  -webkit-backdrop-filter: blur(10px);
+  backdrop-filter: blur(10px);
+  z-index: 5;
+  display: flex;
+  padding: 0 1.6rem;
+  justify-content: center;
+}
+
+.modalWindow {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  gap: 2.4rem;
+  width: max-content;
+  height: 90vh;
+  max-height: max-content;
+  margin: auto;
+  padding: 1.6rem;
+  background-color: white;
+  z-index: 5;
+  overflow-y: scroll;
+}
```

Comment:
- [ ] Your modal window is not responsive in both mobile and desktop versions. The content inside does not fit inside the popup window in some screen sizes. Kindly fix that.
Your design:
Desktop -
<img width="935" alt="image" src="https://user-images.githubusercontent.com/61919617/181069255-73865c02-3bf2-4076-abbb-277ef6454417.png">
Mobile -
<img width="538" alt="image" src="https://user-images.githubusercontent.com/61919617/181069475-e26dd88b-326d-4656-8b18-74db3dd35f29.png">

Reply:
I have made changes to fit the content in the popup